mobility_games/games/wav_musical_cane.py

```diff
@@ -28,17 +17,10 @@ def __init__(self):
         top = RosPack().get_path('mobility_games')
         self.sound_folder = os.path.join(top, 'auditory/sound_files')
         rospy.init_node('audio_feedback')
-        rospy.Subscriber('/tango_pose', PoseStamped, self.process_pose)
-        rospy.Subscriber('/cane_tip', PoseStamped, self.process_cane_tip)
-        rospy.Subscriber('/fisheye_undistorted/tag_detections',
-                        AprilTagDetectionArray,
-                        self.tag_array)
-
-        #   Set up dynamic reconfigure server
-        srv = Server(MusicalCaneConfig, self.config_callback)
 
         #   Set initial values for reconfigure
-        self.mode = rospy.get_param("~mode", "music")
+        self.mode = rospy.get_param("~mode", "sound")
+        self.mode = 'sound'
```

Comment:
Is line 22 even doing anything, considering `self.mode` is being reset immediately afterward?